scripts/run_tests.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# DynamoDB Repository - Integration Tests Runner
+# Assumes DynamoDB Local emulator is already running on port 8000
+
+cd "$(dirname "$0")/.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)